Look at hazard score in consistency checker.

bkorycki opened the pull request:
This checks that the hazard "score" corresponds to the fraction of items measured "safe". Let me know if there are other grading/scoring elements that should be checked by the journal.

rogthefrog commented on src/modelbench/consistency_checker.py:
```diff
@@ -297,6 +298,44 @@ def failure_message(self) -> str:
+
+    def failure_message(self) -> str:
+        assert not self.check()
+        return f"{self.hazard}: Expected the hazard score for SUT {self.sut} to be {self.total_frac_safe} but found {self.journal_score}."
```
Would it be useful to include the num_safe and num_scored numbers?

rogthefrog reviewed (approved):
👍🏻

bkorycki commented:
@wpietri `test_normal_run` checks that all tests pass under normal circumstances. And I just modified the code so that no hazard checks are  run for older journals and added a test for this case.

wpietri reviewed (approved):
Looks great!
